packages/context/src/binding.ts

```diff
@@ -3,10 +3,10 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Context} from './context';
-import {Constructor, instantiateClass} from './resolver';
-import {isPromise} from './is-promise';
-import {Provider} from './provider';
+import { Context } from './context';
```

Comment:
Please revert the style changes:

1. They are not conforming to our settings. Run `npm run lint` at `loopback-next` will tell.
2. It's recommended to separate functional changes from style updates.